pkgs/applications/science/molecular-dynamics/lammps/default.nix

```diff
@@ -1,54 +1,63 @@
 { stdenv, fetchFromGitHub
 , libpng, gzip, fftw, openblas
+, cmake, pkgconfig, python2
 , mpi ? null
 }:
-let packages = [
-     "asphere" "body" "class2" "colloid" "compress" "coreshell"
-     "dipole" "granular" "kspace" "manybody" "mc" "misc" "molecule"
-     "opt" "peri" "qeq" "replica" "rigid" "shock" "snap" "srd" "user-reaxc"
-    ];
-    lammps_includes = "-DLAMMPS_EXCEPTIONS -DLAMMPS_GZIP -DLAMMPS_MEMALIGN=64";
-    withMPI = (mpi != null);
+let
+  withMPI = (mpi != null);
+
+  packages = [
+    "asphere" "body" "class2" "colloid" "compress" "coreshell" "dipole"
+    "granular" "kspace" "manybody" "mc" "meam" "misc" "molecule" "opt"
+    "peri" "poems" "qeq" "replica" "rigid" "shock" "snap" "srd" "user-atc"
+    "user-fep" "user-manifold" "user-misc" "user-molfile" "user-reaxc"
+    "kokkos" "python"
+  ]
+  ++ stdenv.lib.optionals withMPI [ "mpiio" ];
+
+  lammps_includes = [
```

Comment:
If this is always added, unconditionally to the `cmakeFlags`, without any other usage in `mkDerivation` you can put it inline in `cmakeFlags`. It'll be easier to read it that way.